src/js/index.js

```diff
@@ -0,0 +1,64 @@
+let list = document.getElementById("espresso-menu-list");
+let form = document.querySelector("#espresso-menu-form");
+
+// 폼 제출
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  if (form.espressoMenuName.value) {
+    addlist(form.espressoMenuName.value);
+    form.espressoMenuName.value = "";
+  }
```

Comment:
```suggestion
  if (!form.espressoMenuName.value) return
  addlist(form.espressoMenuName.value);
  form.espressoMenuName.value = "";
```

1. Early return을 통한 
2. form.espressoMenuName.value 공백에 대한 이슈 ex)